ios/TextileNode.m

```diff
@@ -614,6 +614,10 @@ - (void)fulfillWithResult:(id)result error:(NSError*)error resolver:(RCTPromiseR
 
 // Couple methods that need to be available from RN and within Obj C
 
+- (NSData*)_dataAtPath:(NSString*)pth error:(NSError**)error {
```

Comment:
You could make the RN exported method call this internal one just for consistency